app/app.go

```diff
@@ -201,6 +206,8 @@ type App struct {
 	upgradeHeight int64
 	// used to define what messages are accepted for a given app version
 	MsgGateKeeper *ante.MsgVersioningGateKeeper
+
+	PacketForwardKeeper *packetforwardkeeper.Keeper
```

Comment:
[follow-up][optional] the fields in this struct don't follow a clear order. We could move these two keeprs into the group below the line:
```
// keepers
```

but it's an extreme nit so nvm